<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="gb_link">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Link Download</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pb_linkDL">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Start Download</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_link">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_list">
       <property name="title">
        <string>List download</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pb_file">
            <property name="text">
             <string>Select File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ind_file">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QCheckBox::indicator{background-color: rgb(0, 0, 0);}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_fileDL">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Start Download</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QProgressBar" name="pBar_download">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ind_count">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
